common/trade_routes/00_wc_south_eastern_kingdoms.txt

```diff
@@ -0,0 +1,91 @@
+azeroth_trade = {
+	wealth = 8  
+	trade_post_display_trigger = {
+		south_eastern_kingdoms_trade_post_trigger = yes
+	}
+
+	modifier = {			# How the trade route modifies the provinces it passes
+		castle_tax_modifier = 0.05
+		city_tax_modifier = 0.03
+		temple_tax_modifier = 0.03
+		tribal_tax_modifier = 0.15
+		trade_route_value = 20
+		local_movement_speed = 0.50
+		tech_growth_modifier = 0.20
+		icon = 4
+	}
+
+	start = {				# Start provinces of trade route
+		1 # Stormwind
+		50 # Zulgurub
```

Comment:
Zul'gurub isn't start province. Otherwise, there would be trade flow from Zul'gurub to Vankeiki.
I believe it's because there's `Kuloke -> Zulgurub` path and it somehow make Stormwind start province of this path.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/65714984-4306e480-e0ad-11e9-8d7f-8b140a9a00b2.png)

</details>